transcriptic/config.py

```diff
@@ -107,7 +107,7 @@ def __init__(
         email=None,
         token=None,
         organization_id=None,
-        api_root="https://secure.transcriptic.com",
+        api_root="https://secure.strateos.com",
```

Comment:
Was a bit concerned about this, but I tested locally without issue and it's fine.

Reply:
Since all the APIs are exposed from the web, we should be fine with this as routing of `strateos` to `transcriptic` is already happening.

With regards to references, sure I agree there may be other instances, but we need to address as-and-when we see them.

Reply:
Yea there are broader decisions to be made about renaming this repository and the python client name itself. Let's make those separately from this PR